scapy/automaton.py

```diff
@@ -873,9 +873,21 @@ def master_filter(self, pkt):
         # type: (Packet) -> bool
         return True
 
-    def my_send(self, pkt):
-        # type: (Packet) -> None
-        self.send_sock.send(pkt)
+    def my_send(self, pkt, iface=None):
```

Comment:
Is there's a use case for sending back a packet on multiple interfaces?

Reply:
Some loop prevention protocols send state update packets on device interfaces when a certain condition occurs on one of the ports.